ecosystem/sep-0006.md

```diff
@@ -102,6 +102,7 @@ SEP-6 lays out many options for how deposit and withdrawal can work. These are r
         * Provide an interface to the user that allows them to view the withdrawal details including the computed fee. The user can confirm and then the wallet will initiate the withdrawal by sending the Stellar payment to the address provided by the issuer.
     * If the anchor responds with an interactive flow, handle it as [described in detail](#3-customer-information-needed-interactive).
     * For now, do not handle the non-interactive or status responses to the `/withdraw` request.
+    * For wallets that enable cross-border payments: exchange rate fluctuations might require withdrawal values to slightly vary from the originally provided `amount`. Anchors are instructed to accept a variation of ±10% between the informed `amount` and the actual value sent to the anchor's Stellar account. The withdrawn amount will be adjusted accordingly.
```

Comment:
make the same type of "Some wallets" change here too and we should be good to go